sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -14,16 +14,12 @@ create, manage, and deploy public and private SSL/TLS certificates
 
 ## Getting started
 ### Install the package
-Install the Azure Key Vault Keys client library for Python with [pip][pip]:
-
+Install this library and [azure-identity][azure_identity] with [pip][pip]:
```

Comment:
Should we say Install Azure KeyVault Keys and Azure Identity instead of "this"?

Reply:
The referent seems clear to me, do you find it ambiguous?

Reply:
I find it somewhat inconsistent - especially since one is a link and the other is not. We are also using the term library and package interchangeable at the moment. What is the concern with making them both links and explicitly state the package name?

Reply:
None, have done.